### ISSUE TITLE
Unable to create custom multi word entity

### ISSUE THREAD

**Issue by matteoc** [insideout10/wordlift-plugin#227]
I've made a quick screen capture video: apparently I am unable to create a multi word entity.
https://dl.dropboxusercontent.com/u/83000/ScreenFlow.mp4

**Comment by cyberandy** [comment]
Hi @matteoc thanks for your precious feedback - the case is clear and we need to fix it asap.

**Comment by mcolacino** [comment]
HI @matteoc first of all thanks a lot for the video: it's the better way to understand what happened :-) The problem here is that WordLift does not allow to create new entities if selected text overlaps an existing annotation as in your example. I think we should fix the message in order to be more clear on that. In these situations simply remove the existing annotation, write again the text and then perform the selection in order to create the new entity. Stay tuned for a quick hotfix with a more meaningful warning message!

**Comment by matteoc** [comment]
@mcolacino looks like I was able to create the entity linked to the phrase that I wanted.
I noticed other issues in the process:
1. although the entity "naming" had already been created, it was not linked in the post anymore. As if wordlift was not remembering the assigned custom entity?
2. when I wanted to link "name your business" to "naming" I went through the exact process of creating a new entity, as if there was no entity there. So I had to specify the name, etc.. I expected to need to do this only once.

**Comment by mcolacino** [comment]
@matteoc 
about 1. ... could you please inspect the browser console and share a screenshot with us?
about 2. ... This is a kind of limitation: WordLift allows you to define alterative labels / titles for existing entities and this allows you to improve the analysis capabilities matching also entities sinonyms. In your case what you're doing it's a **pure not semantic linking** (to define "name your business" as secondary label for your "naming" entity should be a wrong approach indeed). What I suggest in these kind of situation it's simply to link the desired entity page without repeating the new entity creation workflow.

**Comment by matteoc** [comment]
@mcolacino sorry I did not check the console but I've encountered a new issue that I'm describing in the video below https://dl.dropboxusercontent.com/u/83000/ScreenFlow3.mp4
Tomorrow I will respond to your comment more precisely.

**Comment by mcolacino** [comment]
@matteoc this seems to be a double bug:
1. First of all the entity is missing both in local storage and in analysis. *Entity with uri http://data.wordlift.it/wl027/entity/naming is missing both in analysis results and in local storage* and **this is why your annotation is no more colored in the backend** ... 
2. Further, *Annotation name your business [353:371] with id urn:local-text-annotation-7nnidb25q95ozr6ikqi5er4hzj1b6xi4 has no entity matches*: this explain why your annotation is missing at all ... 

The whole bug seems to be now similar to #226 and requires more investigation
I hope to give you more updates tomorrow

**Comment by mcolacino** [comment]
HI @matteoc I go to close this issue, 'couse a new messaging was added in 3.3.2 ... I also opened another dedicated issue #241 to report your last documented bug
